apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.ts

```diff
@@ -366,6 +366,7 @@ export class ExpenseDetailsComponent implements OnInit, OnChanges {
                 this.signatureSrc = ApiUrl + 'company/' + this.companyUniqueName + '/image/' + output.file.response.body.uniqueName;
                 let img = ApiUrl + 'company/' + this.companyUniqueName + '/image/' + output.file.response.body.uniqueName;
                 this.DownloadAttachedImgResponse.push(output.file.response.body);
+                this.imgAttachedFileName = output.file.response.body.name;
```

Comment:
Why this is redundant?

Reply:
API need extra params file name As ledger support only single attached file